packages/components/src/card/src/Card.css

```diff
@@ -1,33 +1,32 @@
 .o-ui-card {
     display: grid;
-    background-color: var(--o-ui-bg-alias-surface);
+    border-radius: var(--hop-shape-rounded-md);
 }
 
 /* VARIANT */
-.o-ui-card-elevated {
-    box-shadow: var(--o-ui-bs-alias-lifted);
-    border-radius: var(--o-ui-br-4);
+.o-ui-card-outline {
+    background-color: var(--hop-neutral-surface);
+    border: 1px solid var(--hop-neutral-border-weak);
 }
 
-.o-ui-card-outline {
-    border: 1px solid var(--o-ui-b-alias-low-break);
-    border-radius: var(--o-ui-br-3);
+.o-ui-card-elevated {
```

Comment:
The elevated version has been created, but what about the **[Interactive](https://www.figma.com/file/zSpuR1wOAT99H6YfD29IRl/%F0%9F%90%B8-Hopper-Design-System?type=design&node-id=436%3A254&mode=design&t=QfqzgF5qenTcyKAh-1)** ?

Reply:
If interactive didn't exist in Orbiter before and there is no need for it, I don't think we need to add it. Same thing for elevated. In igloo, we didn't add anything new, just restyled what existed.

Reply:
Unless I'm wrong and it is a requirement.. Maybe they just used the regular card everywhere before, but we shouldn't anymore.

Reply:
The thing is that Orbit has a Tile for "interactivity" and Vicky is right interactive didn't exist in Orbit no need to add it.

Reply:
Igloo's context is a little different, since the mandate was simply to rebrand the application, since in the future it will be transferred to Hopper. 
 
Orbiter should enable developers to create Workleap. We need to provide them with a design system that matches what's in Figma. We're not talking about removing functionality here, but more about adding missing elements.

Reply:
It still does not makes sense as our Card are more complex that what Workleap will offer. I don't see value in providing an empty div with a box shadow and a padding, any dev can replicate this in a minute.

Reply:
Good to know. I didn't know that we should actually be adding missing elements for Orbiter. 
As for devs to be replicating it, it's true, but if they're doing it over and over again, we want to make sure it's consistent everywhere and that we only have to change it in one place. At least that's what i think.

Reply:
Indeed, every developer has the ability to do this, and what Vicky said is correct. The aim of our work is to standardize these elements, whether they're complex or simple.

Would it be difficult to add this option to the component? If not, I see no reason not to.

Reply:
This is not a small change. We rarely have variants that change the resulting rendered DOM element. In this case, instead of a div, we need to render either a Button or an anchor. As francis said, this use case is handle by Tile and TileLink. I would reskin the Tile component to look like the interactive variant of Card.

Reply:
LGTM!